### DISCUSSION TITLE
Dump Issue to structured format

### DISCUSSION THREAD

**Discussion by henryiii** [PyGithub/PyGithub#1778] [Q&A] [answered]
Quick question. How would I dump an issue to a string? Basically, before pygithub, I converted the json returned to yaml, then dumped it to a file. I'd like to dump an Issue to a similar file, is there a way to do it (looks like it is packable, so maybe that is best, but since it had a json interpretation at one point, I thought there might be a quick way to get back to a readable representation, a dict-like/json interpretation).

**Comment by s-t-e-v-e-n-k** [comment, marked as answer]
All objects have a hidden '._rawData' property that returns the JSON used to construct the object. Like so:

```
>>> import json
>>> repo = g.get_repo("PyGithub/PyGithub")
>>> i = repo.get_issue(1773)
>>> i.title
'The query run error results'
>>> json.dumps(i._rawData)
'{"url": "https://api.github.com/repos/PyGithub/PyGithub/issues/1773", ...
```

**Reply by henryiii** [reply]
Excellent, thank you - that's exactly what I was looking for. Eventually, once I know exactly what I need for later steps, I'll write the code to just add the parts I care about, but for the current development, this is perfect.
